cli2/dependencies.ts

```diff
@@ -0,0 +1,19 @@
+import * as util from "util";
+import * as fs from "fs";
+
+const fsExists = util.promisify(fs.exists);
+const fsReadFile = util.promisify(fs.readFile);
+
+export async function getDependecies(
+  localDependencies: string[]
+): Promise<any[]> {
+  const loadedDependencies = localDependencies.map(async (dependencyPath) => {
+    if (await fsExists(dependencyPath)) {
```

Comment:
`fs.exists` is deprecated and it seems to be causing issues in practice. Please check out the "read (RECOMMENDED)" section here: https://nodejs.org/api/fs.html#fsexistspath-callback. You'll need to change that example to use promises through `promisify` but it should still provide guidance.